src/config.ts

```diff
@@ -458,6 +454,12 @@ export const IBCAssetInfos: {
 		destChannelId: 'channel-2',
 		coinMinimalDenom: 'boot',
 	},
+	{
```

Comment:
Please check that the channel info is correct.

Reply:
Is correct.